blocks/header/header.js

```diff
@@ -283,22 +218,26 @@ export default async function decorate(block) {
   const navMeta = getMetadata('nav');
   const navPath = navMeta || (getLanguage() === 'en' ? '/_drafts/himanshu/nav' : `/${getLanguage()}/nav`);
   const resp = await fetch(`${navPath}.plain.html`);
-
+  const navTreeResp = await fetch(`/nav-tree.json?sheet=${getLanguage()}`);
```

Comment:
Do we cache this  in browser so that we don't end up downloading it on every page load?